test/ng/directive/ngClassSpec.js

```diff
@@ -409,6 +409,78 @@ describe('ngClass', function() {
     expect(e2.hasClass('even')).toBeTruthy();
     expect(e2.hasClass('odd')).toBeFalsy();
   }));
+
+  it('should do value stabilization as expected when one-time binding', inject(function($rootScope, $compile) {
+    element = $compile('<div ng-class=" :: className"></div>')($rootScope);
+    $rootScope.$digest();
+
+    $rootScope.className = 'foo';
+    $rootScope.$digest();
+    expect(element.hasClass('foo')).toBe(true);
+
+    $rootScope.className = 'bar';
+    $rootScope.$digest();
+    expect(element.hasClass('foo')).toBe(true);
+  }));
+
+  it('should do value remove the watcher as expected when one-time binding', inject(function($rootScope, $compile) {
```

Comment:
This test is redundat (the above tests is enough to verify the "one-time-bindness".
There should be also tests for `::{foo: bar}` and `::[...]` though. (I'm not sure the array case is supported by one-time binding, but the object case should be.)